mmv1/openapi_generate/parser.go

```diff
@@ -451,7 +451,12 @@ func buildProperties(props openapi3.Schemas, required []string) []*api.Type {
 
 // Trims whitespace from the ends of lines in a description to force multiline
 // formatting for strings with newlines present
-func trimSpacesFromDescription(description string) string {
+// Also trim "Output only." and "Required." from descriptions as this gets duplicated
+func trimDescription(description string) string {
+	description, _ = strings.CutPrefix(description, "Optional. ")
+	description, _ = strings.CutPrefix(description, "Output only. ")
+	description, _ = strings.CutPrefix(description, "Required. ")
+	description, _ = strings.CutPrefix(description, "Immutable. ")
```

Comment:
Immutable actually doesn't currently get added to descriptions by MMv1 (though arguably it should be IMO - but that might be better in a separate PR)